docs/cloud-native-security/kspm.asciidoc

```diff
@@ -1,2 +1,38 @@
 [[kspm]]
 = Kubernetes security posture management
+
+The Kubernetes Security Posture Management (KSPM) integration allows you to monitor how your Kubernetes clusters' configuration measures up to security benchmarks.
+
+To set up the integration, you'll need to first add it to an agent policy, then deploy the KSPM DaemonSet to the Kubernetes clusters you want to monitor.
+
+[discrete]
+== Set up a KSPM integration
+To install the integration:
+
+* Go to *Security -> Dashboards -> Cloud Posture*
+  * Click *Add a CIS integration*
+  * Click *Add Kubernetes Security Posture Management*.
+  * Name your integration.
+  * Select whether to use the CIS or EKS Benchmarks — use CIS unless you're deploying on EKS.
+  * Select the agent policy where you want to add the integration.
+  * Click *Save and continue*, then *Add Elastic Agent to your hosts*.
+  * The *Add agent* wizard appears, and provides you a DaemonSet manifest `.yaml` with your `Fleet ID`, `Fleet URL`, and other configuration information pre-populated.
+
+In this case, the agent will be added by deploying a DaemonSet on the Kubernetes clusters you wish to monitor. To do this, for each cluster:
```

Comment:
```suggestion
In this case, the {agent} will be added by deploying a DaemonSet on the Kubernetes clusters you wish to monitor. To do this, for each cluster:
```

Reply:
Is this linked to the last step in the previous set of steps? If it is, you can add it "under" the first step by using the `+` sign. This will keep it inline with the previous step so it looks like a continuation of the step, and not a new set of instructions.